extensions/amp-story-audio-sticker/0.1/amp-story-audio-sticker.js

```diff
@@ -1,21 +1,98 @@
 import * as Preact from '#core/dom/jsx';
 import {Layout_Enum} from '#core/dom/layout';
+import {
+  closestAncestorElementBySelector,
+  scopedQuerySelector,
+} from '#core/dom/query';
 
 import {Services} from '#service';
 
 import {CSS} from '../../../build/amp-story-audio-sticker-0.1.css';
-import {Action} from '../../amp-story/1.0/amp-story-store-service';
+import {
+  Action,
+  StateProperty,
+} from '../../amp-story/1.0/amp-story-store-service';
 
 const TAG = 'amp-story-audio-sticker';
 
+/**
+ * The gstatic URL prefix of the sticker assets.
+ * @const {string}
+ */
+const ASSET_URL_PREFIX = 'https://www.gstatic.com/amphtml/stamp/audio-sticker/';
+
+/** Fallback default sticker. It would be used if both of below happen:
+ * 1) no default sticker is specified
+ * 2) the custom sticker data is not provided completely.
+ * @const {string}
+ */
+const FALLBACK_DEFAULT_STICKER = 'cat-sticker';
+
+/** @const {!Object}
+ * List of default stickers.
+ */
+const DEFAULT_STICKERS = {
+  'audio-cloud': {
+    width: '360',
+    height: '233',
+    pretapUrl: ASSET_URL_PREFIX + 'audio-cloud-pretap.png',
+    posttapUrl: ASSET_URL_PREFIX + 'audio-cloud-posttap.png',
+  },
+  'cat-sticker': {
+    width: '282',
+    height: '226',
+    pretapUrl: ASSET_URL_PREFIX + 'cat-sticker-pretap.png',
+    posttapUrl: ASSET_URL_PREFIX + 'cat-sticker-posttap.gif',
+  },
+  'loud-speaker': {
+    width: '380',
+    height: '380',
+    pretapUrl: ASSET_URL_PREFIX + 'loud-speaker-pretap.png',
+    posttapUrl: ASSET_URL_PREFIX + 'loud-speaker-posttap.png',
+  },
+  'tape-player': {
+    width: '260',
+    height: '220',
+    pretapUrl: ASSET_URL_PREFIX + 'tape-player-pretap.png',
+    posttapUrl: ASSET_URL_PREFIX + 'tape-player-posttap.gif',
+  },
+};
+
+/**
+ * The number of milliseconds to wait before hiding the sticker after the story is unmuted.
+ * @const {number}
+ */
+export const HIDE_STICKER_DELAY_DURATION = 4000;
+
 export class AmpStoryAudioSticker extends AMP.BaseElement {
   /** @param {!AmpElement} element */
   constructor(element) {
     super(element);
+
+    /** @private {?../../../extensions/amp-story/1.0/amp-story-store-service.AmpStoryStoreService} */
+    this.storeService_ = null;
+
+    /** @private {boolean} whether to use default sticker. True if any default sticker is specified or there's no custom sticker provided */
+    this.useDefaultSticker_ =
+      this.element.hasAttribute('sticker') ||
+      !this.element.querySelector('amp-story-audio-sticker-pretap') ||
+      !this.element.querySelector('amp-story-audio-sticker-posttap');
```

Comment:
Get the configuration for the "active sticker" here.

```js
this.sticker_ = this.getSticker_();

function getSticker_() {
  const stickerAttr = this.element.getAttribute('sticker');
  if (DEFAULT_STICKERS[stickerAttr]) {
    return DEFAULT_STICKER[stickerAttr];
  }
  if (this.element.querySelector(STICKER_PRETAP_TAG) || this.element.querySelector(STICKER_POSTTAP_TAG)) {
    return null;
  }
  return DEFAULT_STICKERS['cat-sticker']
}
```
Also, is the cat sticker the default?